### ISSUE TITLE
Accessibility:Text area: This TEXTAREA has an id attribute of 'templateExample', which is not unique in the same DOM.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3500]
Description: This TEXTAREA has an id attribute of 'templateExample', which is not unique in the same DOM.
Code snippet:
<textarea style="max-height: 60px;" class="is-warning fd-textarea fdp-textarea-custom-resize ng-untouched ng-pristine ng-valid" aria-disabled="false" placeholder="Start entering detailed description" id="templateExample" wrap="hard" cols="10" rows="2" fd-form-control=""></textarea>

Expected: 
Documents that use markup languages must be well-formed. When documents are not well-formed user agents and/or assistive technology may not correctly parse and render the content to users. Many major user-agents are able to cope well with "bad" markup, however, to be consistent across all platforms and browsers well-formed markup must be used. This is especially important for assistive technology which may or may not be programmed to perform the same level of "coping" that can be found in browsers (which are created by multinational corporations) or assistive technology brands that cost several hundred dollars or more.
Ensure that all page elements have complete start and end tags, elements are nested according to specification, elements do not contain duplicate attributes, and any IDs are unique (except where allowed by the document specification). Mismatched opening and closing quotes, malformed attributes, attributes not space separated, mismatched opening and closing tags, and in some cases, unquoted attributes are also violations.
When documents do not contain a doctype definition the user agent may not be able to determine what grammars the document must conform to. A missing doctype is not a guaranteed failure in cases where it prevents the browser from correctly rendering the document structure it may be a violation. When a doctype is necessary to parse the document structure correctly it must be provided.
The best way to know that a document meets these requirements is to run the document through the W3C's Nu HTML validator. And then review the errors/warnings.

screenshot:

![image](https://user-images.githubusercontent.com/32538291/94652302-6cd59580-0317-11eb-9a7a-7e1a738c40dd.png)
